Common/Transforms/itkAdvancedRigid2DTransform.hxx

```diff
@@ -121,10 +121,10 @@ AdvancedRigid2DTransform< TScalarType >
 {
   // Extract the orthogonal part of the matrix
   //
-  vnl_matrix< TScalarType > p( 2, 2 );
+  vnl_matrix_fixed< TScalarType, 2, 2 > p( 2, 2 );
```

Comment:
While I do think it could be nice to use `vnl_matrix_fixed` here (for style or performance reasons), I wonder, did the original code really cause a build error?

Reply:
Oh, sorry Andras, just now I see your MacOs error message! So it has to do with [the next line](https://github.com/SuperElastix/elastix/blob/5.0.0/Common/Transforms/itkAdvancedRigid2DTransform.hxx#L125), which does:

    p = this->GetMatrix().GetVnlMatrix();

I see now that `itk::Matrix::GetVnlMatrix()` returns a `vnl_matrix_fixed`, and it did so for many years already!

Personally I would strongly prefer combining the two lines of code (construction + assignment), as  in:

    vnl_matrix_fixed< TScalarType, 2, 2 > p = this->GetMatrix().GetVnlMatrix();

Would that be OK to you as well? (If so, `p` might as well be declared `const`!)

Reply:
It is OK for me as long as it compiles on all three platforms.

Reply:
@lassoan I'm sorry, here at github.com/SuperElastix/elastix we only compile elastix + ITK 5.0.1, on the CI (for three platforms). I'll just merge your PR if we can get it compiled here.